### PR TITLE
added collection on route

### PR DESCRIPTION
**Pull request by paulobarcelosbr**

### CODE REVIEW

File: src/components/SweepFlipForm.tsx [hunk 1]
```diff
@@ -49,12 +52,27 @@ function SweepFlipForm({ chainId, activeTab }: FormCardProps) {
...
+  useEffect(() => {
+    const collectionSelected = router?.query.collectionSelected?.toString()
+    if (collectionSelected) {
+      setRouteCollectionSelected(collectionSelected)
+    }
+  }, [router.query])
+
+  useEffect(() => {
+    if (routeCollection.length) {
+      setCollectionData(routeCollection[0])
```
Comment (arthurmr96) [inline src/components/SweepFlipForm.tsx:73]: Why you're getting the first position of an array if the `routeCollection` is a string?

**Review by arthurmr96 — CHANGES_REQUESTED** [review]

File: src/components/SweepFlipForm.tsx (hunk 1 above)
Comment (paulobarcelosbr) [reply src/components/SweepFlipForm.tsx:73]: is  a Array of collection

Comment (arthurmr96) [reply src/components/SweepFlipForm.tsx:73]: Then you should better type the state to be an array, what do you think? So it gets clear that you're dealing with an array

File: src/components/SweepFlipForm.tsx [hunk 2]
```diff
@@ -49,12 +52,27 @@ function SweepFlipForm({ chainId, activeTab }: FormCardProps) {
   const [maxInput, setMaxInput] = useState<number>(1)
   const [isSweepAndFlip, setIsSweepAndFlip] = useState<boolean>(false)
   const [insufficientAmount, setInsufficientAmount] = useState<boolean>(false)
-
+  const [routeCollectionSelected, setRouteCollectionSelected] = useState('')
```
Comment (arthurmr96) [inline src/components/SweepFlipForm.tsx:55]:
```suggestion
  const [routeCollectionSelected, setRouteCollectionSelected] = useState<string[]>([])
```

**Review by arthurmr96 — CHANGES_REQUESTED** [review]

**Review by arthurmr96 — APPROVED** [review]

**Pull request merged**